app/jobs/appointment_confirmed_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+class AppointmentConfirmedNotificationJob < ActiveJob::Base
```

Comment:
Thinking about these job classes, maybe we should just collapse them and pass the `message` as an argument. We could then just `case` over the `message` and invoke the right there. The activity creation and push notification stuff would just work as they're common across all three. WDYT?